Authentication/Login/login.css

```diff
@@ -0,0 +1,261 @@
+@import url(/Utility/utility.css);
+
+.login-page-contents {
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+}
+
+.main-wrapper {
+    flex-grow: 1;
+    flex-direction: row;
+    gap: 3%;
+}
+
+.main-wrapper .image-wrapper {
+    width: fit-content;
+}
+
+.main-wrapper img {
+    width: clamp(18rem, 45%, 25rem);
+    margin: 0;
+}
+
+.inner-icon {
+    padding: 1rem;
+}
+
+.icon {
+    color: var(--color-light-gray);
+    font-size: 2rem;
+}
+
+.inner-icon {
+    color: var(--color-light-gray);
+    font-size: 1.3rem;
+    padding: 1rem;
+}
+
+.login-page-contents .notification {
+    font-size: 1rem;
+    right: -1.2rem;
+}
+
+nav {
+    gap: 8%;
+}
+
+nav .actions {
+    gap: 2rem;
+}
+
+nav .input-box {
+    /* width: fit-content; */
+    min-width: 30%;
+}
+
+nav .input-box input {
+    padding: 1rem 0;
+}
+
+nav h3 {
+    padding: 3%;
+}
+
+.content li,
+.actions {
+    margin: 1rem 0;
+}
+
+.input-box input {
+    padding: 12px;
+}
+
+.input-box {
+    padding: 1rem 0;
+}
+
+.input-box button {
+    padding: 1.1rem;
+    outline: none;
+}
+
+.input-icon-container {
+    flex-wrap: nowrap;
+    border-radius: 0;
+}
+
+nav input::placeholder {
+    font-size: 1.4rem;
+}
+
+.card {
+    padding: 1rem;
+    min-width: 50%;
+}
+
+.card h3 {
+    padding: 1rem 0;
+}
+
+.save-my-info {
+    gap: 8px;
+}
+
+.actions {
+    padding: 0 8px;
+}
+
+.actions span p,
+.save-my-info p {
+    font-size: 1.1rem;
+}
+
+.actions span {
+    color: var(--color-tertiary);
+    border-bottom: 2px solid var(--color-reset);
+}
+
+.save-my-info {
+    font-size: 1.3rem;
+}
+
+button {
+    border-radius: 0;
+    box-shadow: var(--shadow-mild);
+}
+
+.action-btn button {
+    box-shadow: none;
+}
+
+.action-btn button:active {
+    top: 0;
+    border: 2px solid black;
+}
+
+.solid-primary:hover {
+    background-color: var(--color-tertiary-light);
+    color: var(--color-primary)
+}
+
+.transparent button p {
+    text-decoration: underline;
+}
+
+.outline-secondary {
+    padding: 12px 20px;
+}
+
+.icon-with-text {
+    background-color: var(--color-reset);
+}
+
+.icon-with-text p {
+    gap: 1rem;
+    font-size: 1.2rem;
+    ;
+}
+
+.icon-with-text p:hover {
+    border-bottom: 1px solid;
+    transition: 0.4s;
+}
```

Comment:
You can add some type of transition and what is it you want to transition. Do give a look.

Reply:
actually I am still learning about them, will surely add it in future.